### PR TITLE
dev -> staging (v3.1.0)

### PR DESCRIPTION
**Pull request by praveek**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: jitpack.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+# OF ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+#
+
+jdk:
+  - openjdk17
+install:
+   - make ci-publish-maven-local-jitpack
```
Comment (emdobrin) [inline jitpack.yml:16]: do we need this step for media or can we rely on staging snapshots?

Comment (praveek) [reply jitpack.yml:16]: We have added jitpack support to all the extensions.

**Review by kevinlind — APPROVED** [review]

File: Documentation/getting-started.md [hunk 2]
```diff
@@ -10,13 +10,25 @@ Media extension depends on the following extensions:
...
 1. Installation via [Maven](https://maven.apache.org/) & [Gradle](https://gradle.org/) is the easiest and recommended way to get the AEP SDK into your Android app. Add the Mobile Core, Identity, Analytics, and Media extensions to your project using the app's Gradle file:
 
-
-   ```
-   implementation 'com.adobe.marketing.mobile:core:2.+'
-   implementation 'com.adobe.marketing.mobile:identity:2.+'
-   implementation 'com.adobe.marketing.mobile:analytics:2.+'
-   implementation 'com.adobe.marketing.mobile:media:3.+'
-   ```
+    ### Kotlin
```
Comment (emdobrin) [inline Documentation/getting-started.md:13]: nit: use h4 for code samples

File: code/testapp/src/main/java/com/adobe/marketing/mobile/media/testapp/MainActivity.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import androidx.appcompat.app.AppCompatActivity;
+import com.adobe.marketing.mobile.Assurance;
+
+public class MainActivity extends AppCompatActivity {
```
Comment (emdobrin) [inline code/testapp/src/main/java/com/adobe/marketing/mobile/media/testapp/MainActivity.java:22]: nit: the diff is confused and these show as new files, anything we can do to show as updated instead?

**Pull request merged**